app/views/links/history.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= turbo_stream_from @link %>
+<%= render @link %>
+<h1>Past Wallpapers</h1>
+<p>Below is a list of the last <%= pluralize(@past_links.length, "wallpaper")%> set by others for this link. Click any wallpaper to go to see the full size image.</p>
+<div class="past-link-posts past-link-posts--grid">
```

Comment:
I didn't make separate CSS for this page because, honestly, it looks perfectly fine as-is. If I was going to rework the styling on this page, I'd probably also have functional concerns in mind (Button to go to the e6 post instead of the image, display the setter, display the reaction to the image...)

Reply:
Yeah after thinking about it some more, a lot of the layout here is gonna get ripped up anyway once I start adding more functionality to this page. So even though this is barebones, I don't want to get ahead of myself.